ui/src/components/table/TableHeader.jsx

```diff
@@ -71,13 +76,28 @@ function TableHeader({
         );
     };
 
+    const getStatusCount = () => {
+        const enabledRowCount = allFilteredData.filter((item) => !item.disabled).length;
+
+        if (totalElement <= pageSize) {
+            return '';
+        }
+
+        return (
+            <StatusCountWrapper>
+                ({enabledRowCount} of {totalElement} enabled)
```

Comment:
I would simply add a space by `{' '}` or `&nbsp;` instead of styling. Semantically, it should be separated by space, not by padding